.env.example

```diff
@@ -182,6 +189,11 @@ JWT_KEY=
 ## Enables SSL. Note: deprecated
 # SMTP_SECURE=false
 
+###################### Gravatar #######################
+# GRAVATAR_ENABLE=true
+## https://en.gravatar.com/site/implement/images#default-image
+# GRAVATAR_DEFAULT=mp
```

Comment:
I think we should default this to either `404` or `identicon`. I'm leaning more to `404`.

https://en.gravatar.com/site/implement/images#default-image